data/2018-11/agile-tour-bangkok-2018.md

```diff
@@ -0,0 +1,27 @@
+---
+id: gucode-is-cool-201811
+date: "2018-11-02"
+time: "09:00 ~ 17:00"
+location:
+  title: TODO
```

Comment:
No place yet? For now we can use “Bangkok (TBD)” and put Bangkok’s Google Maps URL